content/en/docs/setup/install/istioctl/index.md

```diff
@@ -45,14 +45,16 @@ The simplest option is to install the `default` Istio
 using the following command:
 
 {{< text bash >}}
-$ istioctl install
+$ istioctl install --manifests=manifests/
```

Comment:
This breaks the flow of the doc. The sentence, just above, says this is the `simplest option`, but then passes the `--manifests` arg. The following example, installs the demo profile, without passing `--manifests`, so that seems simpler. 

Also, I think there are other docs that say or imply that all you need is to download `istioctl` to install Istio. Users don't NEED to download the release. This change makes downloading the release a prereq for using `istioctl install`, which sounds bad.

Reply:
Ok, we need to pass `--manifests` everywhere is the primary goal of the PR. The agreement in Environments WG was we would remove the "compiled in" charts. Seems like that will have a bigger docs impact than I originally expected

Reply:
You should probably wait for @richardwxn's PR to merge before making this change because it sounds like the commands he's adding will also need to change: https://github.com/istio/istio.io/pull/7791

Reply:
Or maybe not? He's only doing `istioctl uninstall`

Reply:
uninstall will also need charts to be passed in most cases but that flag is already supported in the subcommand.